private/scripts/helpers.sh

```diff
@@ -1,5 +1,46 @@
 #!/usr/bin/env bash
 
+# Defines some global variables for colors.
+normal=$(tput sgr0)
+bold=$(tput bold)
+red=$(tput setaf 1)
+green=$(tput setaf 2)
+yellow=$(tput setaf 3)
+# Additional styles we aren't currently using.
+# italic=$(tput sitm)
+#blue=$(tput setaf 4)
+#magenta=$(tput setaf 5)
+#cyan=$(tput setaf 6)
+
+# Main function that runs the script.
+function main() {
```

Comment:
nit: define main below functions it calls (style, not required)

Reply:
In my mind, `main` is the thing that makes everything else go, and it's the point of interaction for the script, so to me, it makes more sense at the top because it's the thing that's interacted with first.